torq.q

```diff
@@ -418,7 +418,8 @@ readprocfile:{[file]
 	// exit if no port passed via command line or specified in config
 	if[null[output`port]&0i=system"p";
 		.err.ex[`readprocfile;"No port passed via -p flag or found in ",string[file],". Parameters are host: ", string[output`host], ", proctype: ", string[output`proctype], ", procname: ",string output`procname;1]]; 
-	if[not[output[`port] = system"p"]& 0i = system"p";
+	// for finspace do nothing as port assignment is handled by AWS and not used for connections on our side
+	if[not[.finspace.enabled]&not[output[`port] = system"p"]& 0i = system"p";
```

Comment:
this will already only be true if `0i = system"p"` i.e. no port is set

so if port assignment is handled by AWS, how are we triggering this code? is there no port set automatically in dev clusters? can we instead just add a command line parameter of `-p 0W` if the port number doesn't matter?

Reply:
For non DEV clusters this will not get hit, but the DEV cluster seem to set it's port later and this gets triggered. I'll test adding -p to see if this can be removed